ote_sdk/ote_sdk/usecases/exportable_code/demo/README.md

```diff
@@ -16,15 +16,23 @@ Demo package contains simple demo to get and visualize result of model inference
 
 ## Prerequisites
 * Python 3.8+
+* Git
 
 ## Setup Demo Package
 
-1. Install Python (version 3.8 or higher), [setuptools](https://pypi.org/project/setuptools/), [wheel](https://pypi.org/project/wheel/).
+1. Install Python (version 3.8 or higher).
 
 2. Install the package in the clean environment:
```

Comment:
it would be nice to add steps on how to create this `clean environment`, we usually add such info in our docs, see https://github.com/openvinotoolkit/open_model_zoo/blob/master/tools/accuracy_checker/README.md for example,

1. Python
accuracy checker uses Python 3. Install it first:

Python3, setuptools:
sudo apt-get install python3 python3-dev python3-setuptools python3-pip
Python setuptools and python package manager (pip) install packages into system directory by default. Installation of accuracy checker tested only via virtual environment.

In order to use virtual environment you should install it first:

python3 -m pip install virtualenv
python3 -m virtualenv -p `which python3` <directory_for_environment>
Before starting to work inside virtual environment, it should be activated:

source <directory_for_environment>/bin/activate
Virtual environment can be deactivated using command

deactivate

Reply:
added instructions for creating environment